build.gradle

```diff
@@ -52,7 +53,11 @@ sourceSets {
         output.dir(generatedVersionDir, builtBy: 'generateVersionProperties')
     }
 }
-
+spotless {
```

Comment:
CI does it automatically, maybe we should convert it to dry run and fail PR. Wdyt?